### DISCUSSION TITLE
Airspace Levels

### DISCUSSION THREAD

**Discussion by benniAvia** [Akaflieg-Freiburg/enroute#428] [Polls]
Dear all, so I just discovered Enroute and this is absolutely fascinating. I too fly the DV20 (Katana). I took my first experience with Enroute, and wondered on this: Is there a way to either 

- allow for the font size to be made bigger in the settings, **_OR_**
- when zooming in, zoom the text as well? 

Or is there another way of doing this, e.g. to see the airspace levels?

**Comment by kebekus** [comment]
@benniAvia Your font suggestions is already in our TODO list in a slightly different form, [see this issue](https://github.com/Akaflieg-Freiburg/enroute/issues/409). Implementation will need time, however. We are only a very small team, and are currently overwhelmed with literally thousands of e-mails.

**Comment by benniAvia** [comment]
big thanks, so closing this discussion! 😊
